src/ProjectContents/ModalNewContent.tsx

```diff
@@ -0,0 +1,176 @@
+import { useEffect, useMemo, useState } from "react";
+import { Navigate } from "react-router-dom";
+import dayjs from "dayjs";
+import jsonSchemaToJaamSchema from "../@packages/json-schema-to-jaam-schema";
+import jsonSchemaValidator from "../@packages/json-schema-validator";
+
+import { SelectBox, TextField, useModal } from "../@shared";
+import { FieldTitle } from "../ProjectSchema/FieldTitle";
+import { useContentsState, useSetContentsState } from "./useContentsStore";
+import { useCreateContentMutation } from "./useContentsMutation";
+import * as css from "../ProjectSchema/ModalNewSchema.css";
+
+import type { SchemaList } from "../@types/api";
+import type { SchemaPropertyType } from "../@packages/json-schema-to-jaam-schema/types";
+
+type ModalNewContentProps = {
+  token: string;
+  schemaList?: SchemaList[];
+};
+
+type EditorType = "textfield" | "textarea" | "dateInput";
+
+const editorForType: Record<SchemaPropertyType, EditorType> = {
+  text: "textfield",
+  textarea: "textarea",
+  email: "textfield",
+  link: "textfield",
+  number: "textfield",
+  date: "dateInput",
+  boolean: "textfield",
+};
+
+export function ModalNewContent({ token, schemaList }: ModalNewContentProps) {
+  const [currentSchemaIndex, setCurrentSchemaIndex] = useState<number>(0);
+  const { content, contentsError } = useContentsState();
+  const { setContents } = useSetContentsState();
+  const { closeModal } = useModal();
+
+  if (!schemaList) {
+    return <Navigate to="/error" />;
+  }
```

Comment:
```suggestion
  if (!schemaList || isEmpty(schemaList) {
    return <Navigate to="/error" />;
  }
```

schemaLIst가 비어있는 경우도 검증되면 좋을 것 같습니다!

Reply:
기본 스키마인 asset schema가 추가될거라는 걸 가정하고 해놨는데 validation 좀 더 추가해놓는 게 좋겠네요~ 수정해놓을게요